app/assets/stylesheets/variables.scss

```diff
@@ -1,35 +1,46 @@
-@import 'bootstrap-sprockets';
+@import url(http://fonts.googleapis.com/css?family=Source+Sans+Pro:300,400,700);
+@import "sul-styles";
 
+$gray-base: #000;
 
-$stanfordRed: #990000;
-$cardinalRed: #8C1515;
-$darkRed: #820000;
+$sul-masthead-bg-color: $color-beige-10;
+$sul-masthead-border-color: $color-beige-20;
+$brand-primary: $color-cardinal-red;
 $black: #2e2d29;
-$black80: #585754;
-$beige5: #fbfbf9;
-$beige10: #f2f1eb;
-$beige20: #e9e6df;
-$beige30: #e3dfd5;
-$beige40: #d5d0c0;
-$beige85: #aca68d;
-$beige: #9d9573;
-$gray: #3F3C30;
-$gray90: #565347;
-$darkGray: #696969;
-
-$brand-primary: $cardinalRed;
+$color-gray-80: #ccc;
+
+// Make default a beign button
+$btn-default-color: $black;
+$btn-default-bg: $color-beige-10;
+
+$well-bg: $color-beige-20;
+$exhibit-card-border: $color-gray-80;
+
+$navbar-inverse-bg: $brand-primary;
+$navbar-inverse-border: $color-dark-red;
+$navbar-inverse-color: lighten($gray-base, 86%);
+$navbar-inverse-link-color: lighten($gray-base, 86%);
+
+$sul-footer-bg-color: $color-beige-10;
+$font-family-sans-serif: "Source Sans Pro", "Arial Unicode MS", Helvetica, sans-serif;
+$headings-font-family: "Source Sans Pro", "Arial Unicode MS", Helvetica, sans-serif;
+$sul-link-color-border: $color-pantone-401;
+$nav-link-color: $black;
+
+$panel-default-heading-bg: $color-beige-30;
+$panel-success-heading-bg: #c7d1c5;
+
+@import "bootstrap/variables";
+@import "spotlight/variables";
+@import "spotlight/variables_bootstrap";
+
 $brandbar-height: 35px;
-$navbar-inverse-bg: #101010;
-$navbar-inverse-border: #000;
-$navbar-default-border: #ddd;
-$panel-default-heading-bg: #f7f7f7;
-$navbar-default-link-hover-bg: #eee;
-$navbar-default-link-color: #000;
-$navbar-default-link-active-color: #555;
-$navbar-default-link-active-bg: #ebebeb;
+$brandbar-padding-vertical: (($brandbar-height - $line-height-computed) / 2);
 
-$link-color: $black;
+$home-masthead-height: 119px;
 
+$footer-height: 200px;
+$sul-footer-shadow: 0 4px 8px -8px rgba(0, 0, 0, 0.2) inset;
```

Comment:
Color literals like `rgba(0, 0, 0, 0.2)` should only be used in variable declarations; they should be referred to via variable everywhere else.